packages/vitest/src/integrations/webdriver.ts

```diff
@@ -0,0 +1,26 @@
+import { remote } from 'webdriverio'
+import type { Browser } from 'webdriverio'
+import type { ResolvedConfig } from '../types'
+
+const cachedBrowser: Browser | null = null
+
+export async function openBrowser(config: ResolvedConfig) {
+  if (cachedBrowser)
+    return cachedBrowser
+
+  const browser = await remote({
+    logLevel: 'error',
+    capabilities: {
+      'browserName': config.browser as string,
+      'wdio:devtoolsOptions': { headless: config.headless || !!process.env.CI },
```

Comment:
It should be possible to override it (use `??`). I would personally configure this in the `resolveConfig` function